ovs-p4rt/sidecar/testing/p4info_text.h

```diff
@@ -0,0 +1,2770 @@
+// Copyright 2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef TESTING_P4INFO_TEXT_H_
+#define TESTING_P4INFO_TEXT_H_
+
+// P4Info for ES2K Linux Networking V3.
+constexpr char P4INFO_TEXT[] =
```

Comment:
Good call on uses the text as .h file instead of disk reads.

Reply:
Thanks. 😊